RecoBTag/DeepFlavour/src/sorting_modules.cpp

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
please change the extension to .cc

Reply:
will do

Reply:
done in https://github.com/cms-sw/cmssw/pull/19893/commits/1bc6d3f17e5247c222b56475cebecf4a1e752ec0